examples/SampleApp/src/screens/ChannelScreen.tsx

```diff
@@ -109,10 +111,19 @@ const ChannelHeader: React.FC<ChannelHeaderProps> = () => {
             }
           }}
         >
-          <Avatar
-            {...getChannelPreviewDisplayAvatar(channel, chatClient)}
-            size={40}
-          />
+          {displayAvatar.images ? (
+            <GroupAvatar
```

Comment:
@virdesai need some help here. Not working as expected, images/content overflows the container for some reason.

Reply:
@vishalnarkhede I got this working and pushed it to the v2 design branch so this PR isn't needed anymore